client/views/app/flexTabBar.coffee

```diff
@@ -2,6 +2,7 @@ Template.flexTabBar.helpers
 	active: ->
 		return 'active' if @template is RocketChat.TabBar.getTemplate() and RocketChat.TabBar.isFlexOpen()
 	buttons: ->
+		RocketChat.TabBar.getButtons()
```

Comment:
Is this supposed to be here? I think line 5 doesn't add anything to the function